src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1769,29 +1772,31 @@
       <Field Name="InitializerOpt" Type="InitializerExpressionSyntax?" Null="allow"/>
   </Node>
 
+  <AbstractNode Name="BoundObjectCreationExpressionBase" Base="BoundExpression">
```

Comment:
Would it make sense to define a base for all target-typed expressions?

Reply:
> Would it make sense to define a base for all target-typed expressions?

That might not be a smooth change. For example, ```BoundConvertedSwitchExpression``` doesn't inherit directly from ```BoundExpression```. We might revisit this later, but for now I am comfortable stopping at this level. Especially that this common base will be needed regardless, there is a bunch of abstract properties specific to object creation declared explicitly elsewhere.